microovn/ovn/services.go

```diff
@@ -0,0 +1,84 @@
+package ovn
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"fmt"
+	"slices"
+
+	"github.com/canonical/microcluster/state"
+	"github.com/canonical/microovn/microovn/database"
+	"github.com/canonical/microovn/microovn/node"
+)
+type SrvName string
+
+const (
+	SrvChassis SrvName = "chassis"
+	SrvCentral SrvName = "central"
+	SrvSwitch  SrvName = "switch"
+)
+var ServiceNames = []SrvName{SrvChassis,SrvCentral,SrvSwitch}
+
+func DisableService(s *state.State, service string) error {
+	err := s.Database.Transaction(s.Context, func(ctx context.Context, tx *sql.Tx) error {
+		exists, err := database.ServiceExists(ctx, tx, s.Name(), service)
+		if err != nil {
+			return err
+		}
+		if exists != true {
+			return errors.New("This service is not enabled")
+		}
+		return nil
+	})
+	if err != nil {
+		return err
+	}
+
+	if SrvName(service) == SrvCentral{
+		err = snapStop("ovn-ovsdb-server-nb",true)
+		if err != nil {return err }
+		err = snapStop("ovn-ovsdb-server-sb",true)
+		if err != nil {return err }
+		err = snapStop("ovn-northd",true)
+	}
```

Comment:
Nice!

While there exists a `central` systemd service in addition to the above ones, it is kept as a transitional service to avoid breaking anyone's workflow (see commit 595567eb1d33016a30ef76cb98433250ae4358b0).

For our purposes we should avoid using it, so let's add a `else` branch for the case of `service == SrvCentral`.